src/components/App/CustomAppBar.js

```diff
@@ -48,7 +48,7 @@ const CustomAppBar = (props) => {
 
                 <div style={{ flexGrow: '1' }} />
 
-                <LoadSaveScheduleFunctionality />
+                <ScheduleLoginManager></ScheduleLoginManager>
```

Comment:
Can this be self closing too?
```suggestion
                <ScheduleLoginManager />
```